app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>To-do:</h1>
+
+<ul>
+  <% @tasks.each do |task| %>
+    <li><strong>
+    <%= link_to task.name, task_path(task) %></strong></li>
+    <br>
+    <%= button_to 'Change Completion Status', toggle_complete_path(task), method: :put, data: {confirm: "Are you sure?"} %>
```

Comment:
why are you having them do a confirmation here? Its not wrong, just a bit weird!